package.json

```diff
@@ -59,14 +61,39 @@
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^2.4.0",
     "gh-pages": "^3.1.0",
-    "prettier": "^1.19.1",
+    "lint-staged": "^10.0.8",
+    "prettier": "^2.0.5",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "typescript": "^3.8.3"
+    "stylelint": "^13.3.2",
+    "typescript": "^3.8.3",
+    "husky": "^7.0.0"
   },
   "dependencies": {
     "@cmsgov/design-system": "2.7.1",
     "core-js": "^3.6.4",
     "lodash.uniqueid": "^4.0.1"
-  }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "*.{md,html}": [
+      "prettier --write"
+    ],
+    "*.scss": [
+      "prettier --write",
+      "stylelint --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
I don't know if this block is needed if we're using the `pre-commit` file as well. I thought it was an either/or situation.

Reply:
it's been removed